instance/tests/api/test_openedx_instance.py

```diff
@@ -98,6 +98,48 @@ def assert_active_appserver(self, instance_id, appserver_id):
             )
         return response
 
+    def test_instance_list_efficiency(self):
+        """
+        The number of database queries required to fetch /api/v1/instance/
```

Comment:
@bradenmacdonald Let me be an annoyance a little bit and state that this is not the right way to test that:

* We actually don't care how much queries are exactly performed.
* The test is likely quite fragile, since any change in the Django/OCIM implementation might cause it to break.

What we actually care is the speed of loading - to assert that properly we'd better use  proper load test instead.

This is just me being pedantic - as it is written now is definitely better than having nothing at all and implementing proper load testing might be too expensive at this point (but we might consider this in future)